public/app/plugins/panel/canvas/ConnectionAnchors.tsx

```diff
@@ -14,10 +14,13 @@ type Props = {
 
 export const CONNECTION_ANCHOR_DIV_ID = 'connectionControl';
 export const CONNECTION_ANCHOR_ALT = 'connection anchor';
+const anchorPadding = 3;
+export const connectionAnchorHighlightOffset = 8;
```

Comment:
For these consts I'm thinking we should use the PASCAL_CASE format e.g.
`ANCHOR_PADDING`